client/post-editor/controller.js

```diff
@@ -115,6 +115,7 @@ module.exports = {
 
 		function startEditing( siteId ) {
 			context.store.dispatch( setEditorPostId( postID ) );
```

Comment:
Just a thought - should the reducer for `state.posts.edits` be listening for the `EDITOR_POST_ID_SET` action type and reset itself internally, rather than us calling the reset action explicitly? A possible complication from this is that we might not know that the ID is different, though ideally we're only dispatching this action in cases where the ID has in-fact changed. Or we could track ID in `edits` itself when `EDITOR_POST_ID_SET` is dispatched and compare against it in the reducer.